resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/Constants.scala

```diff
@@ -81,7 +81,7 @@ private[spark] object Constants {
   val EXECUTOR_POD_SPEC_TEMPLATE_FILE_NAME = "pod-spec-template.yml"
   val EXECUTOR_POD_SPEC_TEMPLATE_MOUNTPATH = "/opt/spark/pod-template"
   val POD_TEMPLATE_VOLUME = "pod-template-volume"
-  val POD_TEMPLATE_CONFIGMAP = "driver-podspec-conf-map"
+  val POD_TEMPLATE_CONFIGMAP = "-driver-podspec-conf-map"
```

Comment:
This is a driver pod spec which is irrelevant to your PR title `spark.kubernetes.executor.podNamePrefix`.

If you want to broaden the scope, please revise the PR title and description accordingly.

Reply:
I will make all of the config map length adjustments with some more tests in another PR to minimize this.